static/src/stylesheets/module/facia/_item.scss

```diff
@@ -314,7 +314,9 @@ $fc-item-gutter: $gs-gutter / 4;
     // Override u-responsive-ratio
     // Circumvents Chrome bug with positioning, overflow: hidden and columns
     // https://code.google.com/p/chromium/issues/detail?id=527709
-    overflow: visible;
+    .fc-item--list-compact--media & {
```

Comment:
should go in `stylesheets/module/facia/item-layouts/_fc-item--list-compact--media.scss`